sdv/cag/base.py

```diff
@@ -121,6 +126,31 @@ def transform(self, data):
     def _reverse_transform(self, data):
         raise NotImplementedError
 
+    def _table_as_type_by_col(self, reverse_transformed, table, table_name):
```

Comment:
I need this method to correctly rebase `feature/sdv-branch-single-table-cag` on enterprise.